client-app/src/Components/DonatedItemDetails.tsx

```diff
@@ -89,11 +89,17 @@ const DonatedItemDetails: React.FC = () => {
                             <h2>Donated Item Status</h2>
                         </div>
                         {donatedItem.statuses.map(status => (
-                            <p key={status.id}>
-                                <strong>Status:</strong> {status.statusType} -{' '}
-                                <strong>Modified on:</strong>{' '}
-                                {formatDate(status.dateModified)}
-                            </p>
+                            <div>
+                                <p key={status.id}>
+                                    <strong>Status:</strong> {status.statusType} -{' '}
+                                    <strong>Modified on:</strong>{' '}
+                                    {formatDate(status.dateModified)}
+                                </p>
+                                {status.images.map((image: string, index: number) => (
+                                    <img key={index} src={`data:image/jpeg;base64,${image}`} alt={`Status Image ${index}`} style={{ width: '100px', height: '100px', margin: '5px' }} />
```

Comment:
Can you move these styles to css file to keep things organized. Also increase the image size by 1.5x, its too small now